src/script/components/MessagesList/Message/MessageWrapper.tsx

```diff
@@ -189,6 +190,15 @@ export const MessageWrapper: React.FC<MessageParams & {hasMarker: boolean; isMes
     return entries;
   });
 
+  const handleReactionClick = useCallback(
+    (reaction: ReactionType) => {
+      if (!message.isContent()) {
+        return;
+      }
+      return void messageRepository.toggleReaction(conversation, message, reaction, selfId.id);
+    },
+    [conversation, message, messageRepository, selfId.id],
+  );
```

Comment:
I'd recommend not using `useCallback` and `useMemo` as default but rather when it's really needed. 
In this case it's a pure overhead

```suggestion
  const handleReactionClick = (reaction: ReactionType) => {
      if (!message.isContent()) {
        return;
      }
      return void messageRepository.toggleReaction(conversation, message, reaction, selfId.id);
    };
```